### PR TITLE
CMS-735: Implement FE part of sidebar image settings

### PR DESCRIPTION
**Pull request by Flight**
This PR changes:
- focal_point.js fully rewrited (Raphael.js removed)
- leagcy clipboard.html restored (**please do not review**)
- class.js connected
- base.js updated
- scss improved

### CODE REVIEW

File: filer/private/sass/layout/_main.scss [hunk 1]
```diff
@@ -53,14 +53,30 @@
...
+        position: absolute;
+        z-index: 1;
+        top: 50%;
+        left: 50%;
+        width: 26px;
+        height: 26px;
+        margin: -15px 0 0 -15px;
+        border-radius: 30px;
+        background: rgba(255, 255, 255, .5);
+        border: solid 2px #F00;
```
Comment (dhorelik) [inline filer/private/sass/layout/_main.scss]: `#f00` - lowercase

File: filer/templates/admin/filer/tools/clipboard/clipboard.html [hunk 2]
```diff
@@ -1,150 +1,61 @@
...
-        <ul class="actions-list">
-            <li>
-                <a href="#">
-                    <span class="icon fa fa-trash"></span>
-                    <span class="text">Delete</span>
-                </a>
-        </ul>
+{% for clipboard in user.filer_clipboards.all %}
+    <div class="clipboard">
+        <table style="width: 100%; border: 1px solid #DDDDDD;" xmlns="http://www.w3.org/1999/html"
```
Comment (dhorelik) [inline filer/templates/admin/filer/tools/clipboard/clipboard.html:5]: any way to get rid of inline styling?

File: filer/templates/admin/filer/tools/clipboard/clipboard.html [hunk 3]
```diff
@@ -1,150 +1,61 @@
...
+               xmlns="http://www.w3.org/1999/html">
+            <thead>
+                <tr>
+                    <th colspan="3">{% trans "Clipboard" %}</th>
+                </tr>
+            </thead>
+            <tfoot>
+                <tr>
+                    <td colspan="3">
+                        <form action="{% url 'admin:filer-paste_clipboard_to_folder' %}" method="post" style="display: inline;">
```
Comment (dhorelik) [inline filer/templates/admin/filer/tools/clipboard/clipboard.html:15]: `.form-inline` if it uses bootstrap. Or introduce the css rule if it doesn't

File: filer/templates/admin/filer/tools/clipboard/clipboard.html [hunk 4]
```diff
@@ -1,150 +1,61 @@
...
+                            {% csrf_token %}
+                            <input type="hidden" name="folder_id" value="{{ folder.id }}">
+                            <input type="hidden" name="clipboard_id" value="{{ clipboard.id }}">
+                            <input type="hidden" name="redirect_to" value="{{ current_url }}">
+                            {% if is_popup %}<input type="hidden" name="_popup" value="1">{% endif %}
+                            {% if select_folder %}<input type="hidden" name="select_folder" value="1">{% endif %}
+                            <button class="pastelink" type="submit" {% if folder.is_root or not permissions.has_add_children_permission %} disabled="disabled" style="color: gray;"{% endif %} title="{% trans 'Paste all items here' %}"><span>&lArr;</span>&nbsp;Paste</button>
```
Comment (dhorelik) [inline filer/templates/admin/filer/tools/clipboard/clipboard.html:22]: the line is really too long. Maybe split it somehow?

File: filer/templates/admin/filer/tools/clipboard/clipboard.html [hunk 5]
```diff
@@ -1,150 +1,61 @@
...
+                            <input type="hidden" name="redirect_to" value="{{ current_url }}">
+                            {% if is_popup %}
+                                <input type="hidden" name="_popup" value="1">
+                            {% endif %}
+                            {% if select_folder %}
+                                <input type="hidden" name="select_folder" value="1">
+                            {% endif %}
+                            <button class="deletelink" type="submit" title="{% trans 'Move all clipboard files to' %} &#39;{% trans 'unfiled files' %}&#39; {% trans 'folder' %}"><span>{% trans "Empty Clipboard" %}</span></button>
+                        </form>
+                        {% comment %}
```
Comment (dhorelik) [inline filer/templates/admin/filer/tools/clipboard/clipboard.html:35]: do we need the commented out block?

**Comment by Flight** [comment]
@dhorelik clipboard.html - is an old legacy html file, which will be fully removed soon, no need to review it

**Comment by dhorelik** [comment]
ok, missed the PR comment.. LGTM

**Pull request merged**